src/App.jsx

```diff
@@ -1,107 +1,35 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function getAuthor(authorId) {
+  return usersFromServer.find(user => user.id === authorId)
+  || null;
+}
+
+function getAllPostComments(postId) {
+  return commentsFromServer.filter(comment => comment.postId === postId)
+  || null;
```

Comment:
`null` is not reachable because `filter` always return an array which is always a truth value